scripts-dev/nuke-room-from-db.sh

```diff
@@ -29,7 +29,6 @@ DELETE FROM state_groups WHERE room_id = '$ROOMID';
 DELETE FROM state_groups_state WHERE room_id = '$ROOMID';
 DELETE FROM receipts_graph WHERE room_id = '$ROOMID';
 DELETE FROM receipts_linearized WHERE room_id = '$ROOMID';
-DELETE FROM event_search_content WHERE c1room_id = '$ROOMID';
```

Comment:
I think this should be deleting from `event_search`, rather than being removed altogether

Reply:
I added it as second commit